src/etc/inc/openvpn.inc

```diff
@@ -719,6 +719,15 @@ function openvpn_reconfigure($mode, $settings) {
 				break;
 		}
 	}
+	if ($settings['dev_mode'] === 'tun' && isset($config['unbound']['enable']) && isset($config['unbound']['regovpnclients'])) {
```

Comment:
Why lock this to mode 'tun'? It's also possible to make a tap mode OpenVPN without a bridge that has addresses managed by OpenVPN.